sepal_ui/aoi/aoi_model.py

```diff
@@ -616,8 +619,9 @@ def get_ipygeojson(self) -> GeoJSON:
             f["properties"]["name"] = self.name
 
         # adapt the style to the theme
-        style = json.loads((ss.JSON_DIR / "aoi.json").read_text())
-        style.update(color=color.success, fillColor=color.success)
+        if style is None:
+            style = json.loads((ss.JSON_DIR / "aoi.json").read_text())
+            style.update(color=color.success, fillColor=color.success)
```

Comment:
we set as default color, `color.primary` when adding a layer: https://github.com/12rambau/sepal_ui/blob/aac8985731a4db13acdebfd34150482550a3db53/sepal_ui/mapping/sepal_map.py#L790, in this way, the `Aoi layer` will replace the default one?
In my opinion orange is prettier than green over dark theme.